summaries/consts.py

```diff
@@ -0,0 +1,24 @@
+
+SUMMARY_TYPE_OTHER = 'ot'
```

Comment:
Is there a particular reason for using two-letter combinations as keys? I remember we had a similar discussion when adding enums for KSG positions, where the conclusion was that we were to write it out in full text.

Reply:
No, not really. It is just what I typically use when I program 'choices', so it's likely my muscle memory at work here. I will change it to the more verbose versions that we agreed on.